### PR TITLE
FB Revised

### PR DESCRIPTION
**Pull request by skylooop**
Hi! I have re-implemented FB which is very close to the original implementation (hyperparams, networks architectures for F,B) from [Touati et al 2023](https://arxiv.org/abs/2103.07945).

I run several experiments and indeed proposed changes stabilize overall FB training. Moreover, I checked FB performance without finetuning (as in original paper, without IQL finetuning) and performance is much higher, i.e for some reason finetuning degrades performance after pretraining very drastically. Maybe I missed something, but proposed FB revision was tested on whole ExORL domain, matching reported zero-shot scores in various papers (e.g recent [HILP](https://seohong.me/projects/hilp/))

### CODE REVIEW

File: agents/fb_repr.py [hunk 1]
```diff
@@ -416,40 +392,43 @@ def create(
         network_args = {k: v[1] for k, v in network_info.items()}
 
         network_def = ModuleDict(networks)
-        network_tx = optax.adam(learning_rate=config['lr'])
+        network_tx = optax.chain(
```
Comment (chongyi-zheng) [inline agents/fb_repr.py]: Is the gradient clipping important for performance?
